contrib/NeonVM/number.h

```diff
@@ -16,6 +16,10 @@
 #pragma warning(pop)
 #endif
 
+#ifdef _MSC_VER

```

Comment:
Why is this is number.h, when the only use of strdup is in cell.c? This block should be moved to cell.c.